index.js

```diff
@@ -0,0 +1,95 @@
+const fs = require("fs");
+const { Readable } = require("node:stream");
+
+class APIClient
+{
+  static async login(url, username, password) {
```

Comment:
Could add the dev default login credentials and URL as the defaults here to make it easy to use with a dev instance?

Reply:
Storing the dev creds in the file as default seems like a bad idea. 

Since these are positional arguments, even with a defaults, anybody calling that function would still need to add all the positional arguments anyway (unless we move password to be the first argument, which also seems like a bad idea?)

Reply:
Sorry, I don't mean the dev.browsertrix.cloud instance, I meant a local deployment. So for instance:

`static async login(url="http://localhost:30870", username="admin@example.com", password="PASSW0RD!") { ...`

Now that I'm thinking about it though, this would be more useful if there was a CLI. As a library, I agree it doesn't really make sense, so feel free to ignore!

Reply:
I do like the idea of a CLI interface 👀 
but I'm not sure how useful it'd be / I think it'd be low priority

Reply:
someday maybe!